cfme/base/ui.py

```diff
@@ -39,20 +49,55 @@ class LoggedIn(CFMENavigateStep):
 
     def step(self):
         login_view = self.create_view(LoginPage)
-        login_view.log_in(self.obj.appliance.user)
+        user = self.obj.appliance.user
+        login_view.log_in(user)
+
+
+class ConfigurationView(BaseLoggedInPage):
+    flash = FlashMessages('.//div[starts-with(@id, "flash_text_div")]')
+
+    @View.nested
+    class accordions(View):  # noqa
+
+        @View.nested
+        class settings(Accordion):  # noqa
+            ACCORDION_NAME = "Settings"
+            tree = ManageIQTree()
+
+        @View.nested
+        class accesscontrol(Accordion):  # noqa
+            ACCORDION_NAME = "Access Control"
+            tree = ManageIQTree()
+
+        @View.nested
+        class diagnostics(Accordion):  # noqa
+            ACCORDION_NAME = "Diagnostics"
+            tree = ManageIQTree()
+
+        @View.nested
+        class database(Accordion):  # noqa
+            ACCORDION_NAME = "Database"
+            tree = ManageIQTree()
+
+    @property
+    def is_displayed(self):
+        return False
```

Comment:
Yeh, this bit is something I need to bring up with mfalesni, is_displayed shuld default to False, it's here so that we can assert the view is on screen after an action, the problem is it also means you have to implement them for every one.